sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -12346,7 +12518,7 @@
           },
           "properties": {},
           "calculateServer": true,
-          "calculateValue": "/**\r\n * Part-time is always considered \"independant\".\r\n * Full-time student will be considered \"independant\" if\r\n * parent information does not need to be entered after \r\n * all the questions in the personal information are answered or \r\n * if parent information needs to be entered and student\r\n * \t- has both the parents passed away or; \r\n * \t- is estranged from both of parents. \r\n * Otherwise, student is considered \"dependant\".\r\n */\r\n \r\nconst personalInfoAnswered = !!data.hasDependents &&\r\n  !!data.relationshipStatus &&\r\n  !!data.outOfHighSchoolFor4Years &&\r\n  !!data.fulltimelabourForce &&\r\n  ([\"no\",\"preferNotToAnswer\"].includes(data.youthInCare) || !!data.custodyOfChildWelfare);\r\n\r\nif (data.howWillYouBeAttendingTheProgram === \"Part Time\") {\r\n\tinstance.setValue(\"independant\");\r\n} else if (personalInfoAnswered && !data.showParentInformation) { \r\n\tinstance.setValue(\"independant\");\r\n} else if (data.showParentInformation && \r\n\t[\"bothOfMyHaveParentsHavePassedAwayAndIHaveNoLegalGuardian\",\"estranged\"].includes(data.parentbcResident)) {\r\n\tinstance.setValue(\"independant\");\r\n} else {\r\n\tinstance.setValue(\"dependant\");\r\n}",
+          "calculateValue": "/**\r\n * Part-time is always considered \"independant\".\r\n * Full-time student will be considered \"independant\" if\r\n * parent information does not need to be entered after \r\n * all the questions in the personal information are answered or \r\n * if parent information needs to be entered and student\r\n * \t- has both the parents passed away or; \r\n * \t- is estranged from both of parents. \r\n * Otherwise, student is considered \"dependant\".\r\n */\r\n \r\nconst personalInfoAnswered = !!data.hasDependents &&\r\n  !!data.relationshipStatus &&\r\n  !!data.outOfHighSchoolFor4Years &&\r\n  !!data.fulltimelabourForce &&\r\n  ([\"no\",\"preferNotToAnswer\"].includes(data.youthInCare) || !!data.custodyOfChildWelfare);\r\n\r\nif (data.howWillYouBeAttendingTheProgram === \"Part Time\") {\r\n\tinstance.setValue(\"independant\");\r\n} else if (personalInfoAnswered && !data.showParentInformation) { \r\n\tinstance.setValue(\"independant\");\r\n} else if (data.showParentInformation && (data.parentsDeceased === \"yes\" || data.estranged === \"yes\")) {\r\n\tinstance.setValue(\"independant\");\r\n} else {\r\n\tinstance.setValue(\"dependant\");\r\n}",
```

Comment:
When `(data.parentsDeceased === "yes" || data.estranged === "yes")` then the student is `independant` irrespective of any other condition.